tests/phpunit/StorageWriterTest.php

```diff
@@ -399,14 +489,13 @@ public function testAction(): void
         });
         $app->run();
         ob_end_clean();
```

Comment:
whoa! závan minulýho tisíciletí :-) co to ale poslat ven přes logger?

Reply:
no to prave nechces https://github.com/keboola/php-component/issues/45

Reply:
aha! :+1: